src/UI/class-settings-page.php

```diff
@@ -229,13 +229,16 @@ public function screen_settings( string $screen_settings, WP_Screen $screen ): s
 	 */
 	public function add_help_text(): void {
 		$screen = get_current_screen();
+		if ( null === $screen ) {
+			return;
+		}
```

Comment:
Question: I see in docs that, `get_current_screen()` can return null. Can you explain the thinking behind removing this?

Reply:
Yes, that method can return null. That's why we're adding this check. Otherwise, the subsequent calls to `add_help_tab` would fail if `$screen` is null.

Reply:
Sorry, ignore my comment. For some reason I thought we're removing this instead of adding it.

Reply:
FWIW, WP core doesn't have this check: https://github.com/WordPress/WordPress/blob/270f2011f8ec7265c3f4ddce39c77ef5b496ed1c/wp-admin/index.php#L37-L39

`set_current_screen()` is also called right before the `load-*` actions anyway, so the global `$current_screen` returned from `get_current_screen()` would definitely be set: 
https://github.com/WordPress/WordPress/blob/270f2011f8ec7265c3f4ddce39c77ef5b496ed1c/wp-admin/admin.php#L212-L237